### PR TITLE
Removes some Abductor glands, Nerfs Trauma gland

### PR DESCRIPTION
**Pull request by BurgerLUA**
## About The Pull Request

Nerfs the brain trauma gland so that worst case scenario the gland is curable via surgery instead of a lobotomy.

Removes the following glands from rotation:
- Plasma Vomit Gland
- Slime Vomit Gland
- Spider Vomit Gland
- Random Species Transform Gland

## Why It's Good For The Game

All of these glands cause either a considerable amount of problems unworthy of abductors, honestly.

## Changelog
:cl: BurgerBB
balance: The brain trauma abuductor gland can now be cured via basic brain surgery worst case scenario.
del: Removes Abductor Plasma Vomit Gland, Slime Vomit Gland, Spider Vomit Gland, and the Random Species Transform Gland.
/:cl:

### CODE REVIEW

**Comment by FlamingLily** [comment]
Is there a need to remove the transformation gland?

Actually let me just clarify, does it only transform through human subtypes? Or uh, does it transform you through multiple species.

**Comment by BurgerLUA** [comment]
> Is there a need to remove the transformation gland?
> 
> Actually let me just clarify, does it only transform through human subtypes? Or uh, does it transform you through multiple species.



> Is there a need to remove the transformation gland?
> 
> Actually let me just clarify, does it only transform through human subtypes? Or uh, does it transform you through multiple species.

Species that you can transform into are /datum/species/human, /datum/species/lizard, /datum/species/moth, /datum/species/fly
And it also randomized your appearance completely.

**Comment by SpaceVampire** [comment]
I really don't think the spider gland is that big of deal.
I have also seen people ask for it before as part of some TC trade for their spider characters.
Due to the bonus effect where it makes all of them friendly for the person in question.
I'm fine with the Random Species Transform Gland being removed, as you can't really fix it.
Same with the Brain damage one being tweaked even if it was fixable.
You could maybe leave the Special Trauma with the lobotomy chance, as most of those are actually helpful.
But the rest of the removals seem more unnecessary, having friendly summons is cool.
I do like the idea of only leaving behind organs that at least have some kind of upside to em.
Instead of ones that are only designed to mess with the person in question.

**Comment by YakumoChen** [comment]
Don't fully remove the organs from the game, having them off rotation is enough and organs are used in other places.
I kind of agree with above that plasma and species are the only really critically bad ones, the mob vomit ones are fine as they aren't extremely disruptive and the user gets some protection against their mobs.

File: tgstation.dme [hunk 1]
```diff
@@ -1718,11 +1718,7 @@
 #include "code\modules\antagonists\abductor\equipment\glands\electric.dm"
 #include "code\modules\antagonists\abductor\equipment\glands\heal.dm"
 #include "code\modules\antagonists\abductor\equipment\glands\mindshock.dm"
-#include "code\modules\antagonists\abductor\equipment\glands\plasma.dm"
 #include "code\modules\antagonists\abductor\equipment\glands\quantum.dm"
-#include "code\modules\antagonists\abductor\equipment\glands\slime.dm"
-#include "code\modules\antagonists\abductor\equipment\glands\spider.dm"
-#include "code\modules\antagonists\abductor\equipment\glands\transform.dm"
```
Comment (AlinaStarkova) [inline tgstation.dme:1725]: dont remove these files, you may cause issues later down the line if these organs are used elsewhere, simply commenting out the organs from being in rotation are fine as they are, removing the organs completely is just unnesscary

**Comment by Krysonism** [comment]
Getting abducted is not supposed to be a nice thing.

**Comment by softcerv** [comment]
> 
> 
> Getting abducted is not supposed to be a nice thing.

It shouldn't be, but at the same time a lot of the things that abductors can do to people, especially the random species gland, have the potential to ruin someone's round. Especially in a more roleplay focused server with longer rounds, where being abducted is usually worse than death. I still think they should be kept in the code for sure, but not part of the abductor kit.

**Comment by BurgerLUA** [comment]
Closing this PR. If you want this PR merged, merge the one below instead. I'm tired of having to settle for half measures.

https://github.com/Skyrat-SS13/Skyrat-tg/pull/7293

**Comment by AlinaStarkova** [comment]
> 
> 
> Closing this PR. If you want this PR merged, merge the one below instead. I'm tired of having to settle for half measures.
> 
> #7293

its not a half measure to just not delete the files that contain the code, disabling them in rotation was fine, but completely removing the files for them entirely was overboard, you dont need to remove the files to remove them from rotation, hence keeping the files in wouldnt be a "half measure"

**Comment by BurgerLUA** [comment]
> > Closing this PR. If you want this PR merged, merge the one below instead. I'm tired of having to settle for half measures.
> > #7293
> 
> its not a half measure to just not delete the files that contain the code, disabling them in rotation was fine, but completely removing the files for them entirely was overboard, you dont need to remove the files to remove them from rotation, hence keeping the files in wouldnt be a "half measure"

I'm referring to the half measures being that the other initial changes were not accepted.

**Pull request closed without merging**